include/znc/User.h

```diff
@@ -150,6 +150,7 @@ class CUser {
     void SetTimestampFormat(const CString& s) { m_sTimestampFormat = s; }
     void SetTimestampAppend(bool b) { m_bAppendTimestamp = b; }
     void SetTimestampPrepend(bool b) { m_bPrependTimestamp = b; }
+    void SetBuiltinAuthDisabled(bool b) { m_bBuiltinAuthDisabled = b; }
```

Comment:
Let's don't have negation in setting names. Not double negation is not more confusing for human mind.

Reply:
Is "BuiltinAuthEnabled" (defaulting to true) okay instead?

Reply:
Thinking about it a little more, maybe "OnlyModulesAuth" (still defaults to false) is more descriptive... thoughts?